src/lib/components/button/button.module.scss

```diff
@@ -62,16 +64,5 @@
   text-align: left;
   font-size: 0.8rem;
   display: block;
-  margin-top: 1rem;
-}
-
-.linkButton {
-  padding: 1rem 2.5rem;
-  
-  @media screen and (min-width: 52em) {
-    padding: 0.7rem 2.2rem;
```

Comment:
Sorry just a detail and question really (to know how it works), some margins/paddings use rem and others em, I thought rem was just for fonts and em for the rest, is that correct? Apart from that everything seems okey to me!

Reply:
It is a good question, I have the same question...I didn't update styles here, I think rem should be used for fonts, but @lulen11 is the right person to answer that, haha